src/encoded/static/components/motifs.js

```diff
@@ -158,12 +158,27 @@ export class MotifElement extends React.Component {
         const targetList = element.targets;
         const footprintList = {};
         const pwmList = {};
+
+        /*
+        Element {
+            accessions [ ID ]
+            biosamples [ Label ]
+            datasets [ URL ]
+        }
+        */
         element.datasets.forEach((d, dIndex) => {
+            // (URL, Index)
```

Comment:
The comment makes this much clearer. It would have been nice if they had named `d` and `dIndex` more clearly in the beginning. I initially configured igvf-ui to disallow single-character variable names, but it caused too many issues with npm packages.